portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,60 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <style>
+body {
+  background-image: url('images/white-cat-1.png');
+}
+</style>
```

Comment:
nit: Can you indent </style> to match the opening tag (by adding 4 spaces before it). And also indent body and the content under it.  Everything that is nested inside an object above it, should be indented whatever you indent size, from the items above it.

It this file, that is 2 spaces, because that is what everything else is going.

You may not think this is important, but it actually makes things much easier to read, which is really important.

("nit" means some small little thing, that isn't that important, but is good as good hygene, so everything looks good).